### PR TITLE
Fix for handling MIDI channels in the bidirectional converter example

### PR DESCRIPTION
**Pull request by YuuichiAkagawa**
Fix for handling MIDI channels in the bidirectional converter example (#586)

### CODE REVIEW

File: examples/USBH_MIDI/bidirectional_converter/bidirectional_converter.ino [hunk 1]
```diff
@@ -87,6 +88,9 @@ void loop()
           //SysEx is handled by event.
           break;
         default :
+          if( msg[0] < 0xf0 ){
+            msg[0] |= MIDI.getChannel() - 1;
+          }
```
Comment (Lauszus) [inline examples/USBH_MIDI/bidirectional_converter/bidirectional_converter.ino:93]: Could you add a comment explaining why this code is needed, so people less familiar with MIDI (including me) would be able to understand it?

**Review by Lauszus — CHANGES_REQUESTED** [review]

File: examples/USBH_MIDI/bidirectional_converter/bidirectional_converter.ino (hunk 1 above)
Comment (YuuichiAkagawa) [reply examples/USBH_MIDI/bidirectional_converter/bidirectional_converter.ino:93]: - msg [0] is the MIDI status byte.
- Lower 4 bits of the status byte is the channel number.
- Status byte 0xFn is a system message.
- System messages do not have a channel number.

I forgot to set the channel number.
The getChannel() function in the Arduino MIDI library returns 1-16 as channel number.
https://fortyseveneffects.github.io/arduino_midi_library/a00033.html#ga0d2a0037fd9e4b06cafe7b2388451767
However, the channel number of raw MIDI messages is zero origin.

Comment (Lauszus) [reply examples/USBH_MIDI/bidirectional_converter/bidirectional_converter.ino:93]: Okay. Can you add that comment to the example? Then I'll merge it :)

**Review by Lauszus — APPROVED** [review]

**Comment by Lauszus** [comment]
Thanks for the PR. It's merged into master now.

**Pull request merged**